### PR TITLE
Docs Update - README & Overview

### PR DESCRIPTION
**Pull request by viragmody**
@bkchr

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -1,101 +1,98 @@
...
-the notification the so-called candidate message. This candidate message is issued by a Parachain
-validator after approving a block. This proof of possible inclusion prevents spamming other collators
-of the network with useless blocks.
-The Collator joins the relay-chain network for two reasons. First, the Collator uses it to send the
-Parachain blocks to the Parachain validators. Secondly, the Collator participates as light/full-node
-of the relay chain to be informed of new relay-chain blocks. This information will be used for the
-consensus and the block production logic.
+When compiled into a WASM binary and registered with a set of Validator Nodes, the substrate node can join as a Parachain. It does so primarily through a `validate_block` macro call implemented as a Cumulus pallet, which takes the `Block` and `Executive` type. `Executive` is a declared type that dispatches incoming calls to the correct module at runtime; it is an orchestration component—more on `Block` below.
+
+## Consensus
```
Comment (NukeManDan) [inline docs/overview.md:33]: The remainder of this doc, while a _great_ initiative, I think I see a few factual errors. I would propose be removed completely in favor of the official documentation that is formally maintained:

- https://wiki.polkadot.network/docs/en/learn-parachains
- https://wiki.polkadot.network/docs/en/learn-parachains#resources (for blogs with deeper tech details)

What from here should be migrated into those?

**Review by NukeManDan — CHANGES_REQUESTED** [review]
Good effort here! 😀 

I think it may be best to actually migrate this _out_ on the  repo docs into formal docs sites I mentioned:
https://wiki.polkadot.network/docs/en/learn-parachains
https://wiki.polkadot.network/docs/en/learn-parathreads#parachain-vs-parathread
https://substrate.dev/cumulus-workshop/

File: docs/overview.md [hunk 2]
```diff
@@ -1,101 +1,98 @@
...
-Parachain's state transition and also provides interfaces for the Parachain to send and receive
-messages of other Parachains.
+### Why would I use Cumulus?
+Cumulus allows runtimes built using Substrate [FRAME](https://substrate.dev/docs/en/knowledgebase/runtime/frame) to interface with Parachain validators and to join a network secured by a [Relay Chain](https://substrate.dev/docs/en/knowledgebase/getting-started/glossary#relay-chain). 
+
+This distinction is essential as runtimes created with purely Substrate FRAME are not Parachains and cannot join a Polkadot network. Parachains can implement substrate FRAME, but FRAME is a more flexible set of tools for custom and upgradeable blockchains. Cumulus is a wrapper around a Substrate node that extends an interface to Parachain Validators, opening a direct channel to a network secured by its Relay Chain. 
+
+Cumulus lets developers make Parachains using Substrate. Use cases could include:
+- Make a Polkadot-compatible Parachain from genesis.
+- Create a sandbox blockchain with Substrate and connect to the Polkadot network when ready.
```
[the remark this answers is not in the document]
Comment (viragmody) [reply docs/overview.md:15]: Used it colloquially. Could also say private chain.

File: docs/overview.md (hunk 1 above)
Comment (viragmody) [reply docs/overview.md:33]: I wrote this as part of a job application for Parity. It never went through a technical review process.
If you want to point out errors, I can make the changes necessary.

As for where it gets hosted, totally your call.

**Comment by NukeManDan** [comment]
this PR is quite stale, but content can be used as reference for an update latter.

**Pull request closed without merging**